include/aspect/postprocess/interface.h

```diff
@@ -226,10 +230,33 @@ namespace aspect
          * if one of them has the desired type specified by the template
          * argument. If so, return a pointer to it. If no postprocessor is
          * active that matches the given type, return a NULL pointer.
+         * @deprecated: Use has_matching_postprocessor() and
```

Comment:
add an empty line here to make sure we don't actually wrap the whole paragraph. I don't think you need the colon.